packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -320,8 +320,9 @@ def test_dcos_diagnostics_report(dcos_api_session):
         assert len(report_response['Nodes']) > 0
 
 
-@pytest.mark.parametrize('use_legacy_api',
-                         [False, pytest.param(True, marks=pytest.mark.xfail("config.getoption('--windows-only')"))])
+@pytest.mark.parametrize('use_legacy_api', [
+    pytest.param(False, marks=pytest.mark.xfail("config.getoption('--windows-only')"), reason="D2IQ-65964"),
```

Comment:
Please mark it as `strict`

Reply:
it's strict and premissive

Reply:
I don't know how to do that. I agree the best will be to xfail only on EE but I think it's fine for now.

Reply:
No, I meant make `xfail` fail if the test succeeds. It's not that important as we will tackle it next week.